BUILDING.md

```diff
@@ -89,7 +92,7 @@ Depending on host platform, the selection of toolchains may vary.
 
 #### Windows
 
-* Visual Studio 2017 or the Build Tools thereof
+* "Visual Studio 2017" or the "Build Tools" thereof, on a 64 bit host.
```

Comment:
I should have been more explicit: please don't product names in quotes.

Reply:
We actually don't test with the Build Tools package anymore. And we only use Windows SDK 10.